src/QirRuntime/lib/QIR/intrinsicsMath.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) Microsoft Corporation.
```

Reply:
Done (including for my files added to tests).
Where can I get more info about this? (to know in the future what is recommended or not recommended)